src/main/java/utils/configurations/Configurations.java

```diff
@@ -0,0 +1,36 @@
+package utils.configurations;
+
+import com.google.common.collect.ImmutableMap;
+import onlineservices.Service;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import services.WindowControl.WindowControlService;
+import utils.filereaders.CsvFileReader;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class Configurations {
```

Comment:
overall this will not work, you need to add @ComponentScan to the Main class -> @ComponentScan(basePackages = "onlineservices")

Reply:
you forgot this